### PR TITLE
GSdx: Add missing CRC id.

### PR DESCRIPTION
**Pull request by Al0ner**
GSdx: Add missing CRC id. Add TombRaiderUnderworld RU.

### CODE REVIEW

File: plugins/GSdx/GSCrc.cpp [hunk 1]
```diff
@@ -180,6 +180,8 @@ CRC::Game CRC::m_games[] =
 	{0x6BA2F6B9, ResidentEvil4, EU, 0},
 	{0x60FA8C69, ResidentEvil4, JP, 0},
 	{0x5F254B7C, ResidentEvil4, KO, 0},
+	{0x7C48D927, ResidentEvilCodeVeronicaX, RU, 0}, // unofficial RU-version
+	{0x546C846B, RuleOfRose, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: These two shouldn't be added. There isn't any code that looks up those specific games.

File: plugins/GSdx/GSCrc.cpp [hunk 2]
```diff
@@ -197,6 +199,7 @@ CRC::Game CRC::m_games[] =
 	{0x652050D2, Tekken5, US, 0},
 	{0xEA64EF39, Tekken5, KO, 0},
 	{0x9E98B8AE, IkkiTousen, JP, 0},
+	{0x8191D10A, IndigoProphecy, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Ditto.

File: plugins/GSdx/GSCrc.cpp [hunk 3]
```diff
@@ -313,6 +316,9 @@ CRC::Game CRC::m_games[] =
 	{0x615EA2DB, Sly2, JP, 0}, // Kaitou Sly Cooper 2
 	{0xFDA1CBF6, Sly2, EU, 0},
 	{0x15DD1F6F, Sly2, NoRegion, 0},
+	{0x1E33F427, SilentHillShatteredMemories, RU, 0}, // unofficial RU-version
+	{0xEDF02195, SilentHill2, RU, 0}, // unofficial RU-version
+	{0x51CED27D, SilentHill3, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Ditto, all 3.

File: plugins/GSdx/GSCrc.cpp [hunk 4]
```diff
@@ -371,12 +377,14 @@ CRC::Game CRC::m_games[] =
...
 	{0x8E214549, TombRaiderUnderworld, EU, 0},
 	{0x618769D6, TombRaiderUnderworld, US, 0},
+	{0x8E265148, TombRaiderUnderworld, RU, 0}, // unofficial RU-version
 	{0xB639EB17, TombRaiderAnniversary, US, 0}, // Also needed for automatic mipmapping
 	{0xB05805B6, TombRaiderAnniversary, JP, 0}, // cutie comment
 	{0xA629A376, TombRaiderAnniversary, EU, 0},
 	{0xBC8B3F50, TombRaiderLegend, US, 0}, // cutie comment
 	{0x365172A0, TombRaiderLegend, JP, 0},
 	{0x05177ECE, TombRaiderLegend, EU, 0},
+	{0x4C380F9B, TotalOverdose, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Ditto.

File: plugins/GSdx/GSCrc.cpp [hunk 5]
```diff
@@ -427,6 +435,7 @@ CRC::Game CRC::m_games[] =
 	{0x086D198E, Kunoichi, CH, 0},
 	{0x3B470BBD, Kunoichi, EU, 0},
 	{0x6BA65DD8, Kunoichi, KO, 0},
+	{0x9AC63A2E, Kuon, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Ditto.

File: plugins/GSdx/GSCrc.cpp [hunk 6]
```diff
@@ -485,6 +494,7 @@ CRC::Game CRC::m_games[] =
 	{0x59683BB0, DeathByDegreesTekkenNinaWilliams, EU, 0},
 	{0x5B659BED, Grandia3, JP, 0},
 	{0x5B657DAD, Grandia3, US, 0},
+	{0x1E1DC2EF, GrandTheftAutoViceCity, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Ditto.

File: plugins/GSdx/GSCrc.cpp [hunk 7]
```diff
@@ -543,8 +553,10 @@ CRC::Game CRC::m_games[] =
 	{0x230CB71D, SoulReaver2, US, 0},
 	{0x6F991F52, SoulReaver2, JP, 0},
 	{0x6D8B4CD1, SoulReaver2, EU, 0},
+	{0x60F56E8F, SoulReaver2, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Small nitpick, I code comments to begin with capital letters.

File: plugins/GSdx/GSCrc.cpp [hunk 8]
```diff
@@ -543,8 +553,10 @@ CRC::Game CRC::m_games[] =
...
 	{0x728AB07C, LegacyOfKainDefiance, US, 0},
 	{0xBCAD1E8A, LegacyOfKainDefiance, EU, 0},
+	{0x018AC37C, LegacyOfKainDefiance, RU, 0}, // unofficial RU-version
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: Same as Soul Reaver request.

File: plugins/GSdx/GSCrc.cpp [hunk 9]
```diff
@@ -395,7 +403,7 @@ CRC::Game CRC::m_games[] =
 	{0x79C952B0, DevilMayCry3, JP, 0}, // SE
 	{0x7F3DDEAB, DevilMayCry3, JP, 0},
 	{0x05931990, DevilMayCry3, KO, 0},
-	{0x4AD36D59, DevilMayCry3, RU, 0},
+	{0x4AD36D59, DevilMayCry3, RU, 0}, // same CRC as RU
```
Comment (lightningterror) [inline plugins/GSdx/GSCrc.cpp]: It's already RU region.

Comment (Al0ner) [reply plugins/GSdx/GSCrc.cpp]: And what about the line {0x8E265148, TombRaiderUnderworld, RU, 0}?
In the RU version, CRC-hacks are not applied to it, and glitches are not fixed.

Comment (lightningterror) [reply plugins/GSdx/GSCrc.cpp]: That one is fine just change the first letter of the code comment to use capital letter.

Comment (Al0ner) [reply plugins/GSdx/GSCrc.cpp]: That's right?

Comment (lightningterror) [reply plugins/GSdx/GSCrc.cpp]: I don't see anything different other than the PR title.

You need to make the changes in the file and push them.

**Pull request closed without merging**